src/qml/qmlwaveformoverview.h

```diff
@@ -58,7 +58,7 @@ class QmlWaveformOverview : public QQuickPaintedItem {
   signals:
     void playerChanged();
     void channelsChanged(mixxx::qml::QmlWaveformOverview::Channels channels);
-    void rendererChanged(mixxx::qml::QmlWaveformOverview::Renderer renderer);
+    void rendererChanged(Renderer renderer);
```

Comment:
Is this change needed? I'm confused why `channelsChanged` would need it too if yes. Any idea?

Reply:
@acolombier They differ in property type, so likely related to that:
```c++
Q_PROPERTY(Channels channels READ getChannels WRITE setChannels NOTIFY channelsChanged)
Q_PROPERTY(Renderer renderer MEMBER m_renderer NOTIFY rendererChanged)
```